src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -180,6 +180,13 @@ <h1 class="subheader__title" property="name">{% render_model current_page "title
                                 {% if open_visible_runs|length == 0 %}
                                     <div class="course-detail__row course-detail__runs course-detail__runs--open">
                                         <div class="course-detail__empty">{% trans "No open course runs" %}</div>
+                                        {% is_feature_enabled 'JOANIE_WISHLIST' as joanie_wishlist_enabled %}
```

Comment:
Can you update the `env.d/dev.dist` and `env.d/dev-ssl.dist` files to add enable this flag by default for dev ?

Reply:
My bad !